examples/plot_grid_search_census.py

```diff
@@ -143,9 +151,11 @@
 # We are using this metric for the sake of simplicity; in general, the appropriate fairness
 # metric will not be obvious.
 
-sweep = GridSearch(LogisticRegression(solver='liblinear', fit_intercept=True),
-                   constraints=DemographicParity(),
-                   grid_size=71)
+sweep = GridSearch(
+    LogisticRegression(solver="liblinear", fit_intercept=True),
+    constraints=DemographicParity(),
+    grid_size=30,
```

Comment:
Small point, but if this is an odd number, then the grid includes an unmitigated model.